src/components/AgenciesPage/types/Agencies.ts

```diff
@@ -0,0 +1,14 @@
+export interface Agency {
+  id: number;
+  name: string;
+  type: string;
+  administrator: string;
+  country_code: string;
+  launchers: string;
+  image_url: string;
+  spacecraft?: string;
+  [key: string]: string | undefined | number;
```

Comment:
can't use undefined

Reply:
the way how fix it: https://stackoverflow.com/questions/61431397/how-to-define-typescript-type-as-a-dictionary-of-strings-but-with-one-numeric-i